### PR TITLE
Feature/workouts

### PR DESCRIPTION
**Pull request by cyrilbo**

### CODE REVIEW

File: src/navigation/index.tsx [hunk 1]
```diff
@@ -55,7 +46,7 @@ const TabNavigator = createBottomTabNavigator(
           case 'Home':
             iconName = `ios-home`;
             break;
-          case 'Configurator':
+          case 'Workouts':
             iconName = `ios-fitness`;
             break;
```
Comment (MattAgn) [inline src/navigation/index.tsx]: put directly in workoutNavigator

File: src/pages/Workouts/Workouts.tsx [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
-
-export const Workouts: FunctionComponent<WorkoutsProps> = () => {
+export const Workouts: React.FC<NavigationStackScreenProps> = ({ navigation }) => {
   return (
     <Container>
       <RoundButtonContainer>
-        <RoundButton onPress={() => Alert.alert('Add a workout')} />
+        <RoundButton
+          onPress={() => {
+            console.log(navigation);
```
Comment (MattAgn) [inline src/pages/Workouts/Workouts.tsx]: delete

File: src/pages/Workouts/Workouts.tsx [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
+      name: 'Leg Day',
+      muscleGroups: [MuscleGroup.Legs, MuscleGroup.Calves],
+      id: 124,
+      exercises: [],
+    },
+  ];
+
   return (
     <Container>
+      <FlatList data={DATA} renderItem={({ item }) => <WorkoutItem {...item} />} />
```
Comment (MattAgn) [inline src/pages/Workouts/Workouts.tsx]: extract renderFunction

Comment (cyrilbo) [reply src/pages/Workouts/Workouts.tsx]: done

File: src/pages/Workouts/Workouts.tsx (hunk 2 above)
Comment (cyrilbo) [reply src/pages/Workouts/Workouts.tsx]: done

File: src/navigation/index.tsx (hunk 1 above)
Comment (cyrilbo) [reply src/navigation/index.tsx]: done

**Pull request merged**